nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/operations/SNMPTrapReceiver.java

```diff
@@ -46,7 +47,7 @@ public void processPdu(final CommandResponderEvent event) {
         final PDU pdu = event.getPDU();
         if (isValidTrapPdu(pdu)) {
             final ProcessSession processSession = processSessionFactory.createSession();
-            final FlowFile flowFile = createFlowFile(processSession, pdu);
+            final FlowFile flowFile = createFlowFile(processSession, pdu, event.getPeerAddress());
```

Comment:
Really nitpick but instead of adding an argument, isn't it better to just use the event object to pass as an argument of the private method?

Reply:
Just updated to pass just the event.